### PR TITLE
Policy Authz Post

### PR DESCRIPTION
**Pull request by leebrandt**
## Description:
- Blog post on Policy-Based Authorization in ASP.NET Core

### CODE REVIEW

File: _source/_posts/2018-05-11-policy-based-authorization-in-aspnet-core.md [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+}
+<h2>@ViewData["Title"]</h2>
+<h3>@ViewData["Message"]</h3>
+
+<p>This is only for the bourbon.com folks<p>
+```
+
+### Check Your Work
+
+Now when you fire up the application, log in as a user who has a "bourbon.com" email address, and navigate to the bourbon route, you will be granted access to the bourbon.. page. For users that _don't_ have the right email address domain, you are routed to an access denied route. You could send them to a special unauthorized page that gives them more choices to log out and log in with a user with the proper privileges.
```
Comment (mraible) [inline _source/_posts/2018-05-11-policy-based-authorization-in-aspnet-core.md]: I'm guessing "bourbon.." should be "bourbon.com"?

File: _source/_posts/2018-05-11-policy-based-authorization-in-aspnet-core.md [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+
+{% img blog/policy-based-authz-core/bourbon-is-life.png alt:"Application running on restricted page" width:"800" %}{: .center-image }
+
+As you can see, this new approach to authorization in ASP.NET Core allows you to separate your requirements, making them easy to combine in different ways. Combining these requirements into custom policies allow you to achieve a scalable approach to fine-grained permissions!
+
+## Learn More About Securing Your ASP.NET Core Applications
+
+Didn't get enough ASP.NET Core goodness? Check out our other cool posts about [ASP.NET Core with Angular](/blog/2018/04/26/build-crud-app-aspnetcore-angular), [Token Authentication in ASP.NET Core 2](/blog/2018/03/23/token-authentication-aspnetcore-complete-guide), and a more in-depth look at [roles-based authorization in ASP.NET Core](/blog/2017/10/04/aspnet-authorization). Don't forget to checkout [Microsoft's Documentation on policy-based authorization](https://docs.microsoft.com/en-us/aspnet/core/security/authorization/policies?view=aspnetcore-2.1)!
```
Comment (mraible) [inline _source/_posts/2018-05-11-policy-based-authorization-in-aspnet-core.md]: I would add a "too" at the end of the last sentence.

File: _source/_posts/2018-05-11-policy-based-authorization-in-aspnet-core.md (hunk 1 above)
Comment (leebrandt) [reply _source/_posts/2018-05-11-policy-based-authorization-in-aspnet-core.md]: Nope it's the bourbon page. Subtle attempt at humor. You have access to the bourbon..    page :)

Comment (mraible) [reply _source/_posts/2018-05-11-policy-based-authorization-in-aspnet-core.md]: With two periods, it looks like a typo. Should it be three?

**Pull request merged**